### ISSUE TITLE
Ability to choose angle for mines and dynamite confuses

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#457]
```
Many times I saw people accurately choosing angle when placing mines or 
dynamite, wasting their turn time. Why was it allowed anyway? Should be removed 
IMHO.
```

Original issue reported on code.google.com by `unC0Rr` on 31 Oct 2012 at 12:04

**Comment by GoogleCodeExporter** [comment]
```
not too sure. I expect it is more simply a side effect of graphics having been 
added and the code never having had a place to indicate whether gears needed an 
angle before that.  min/max angle of 0 indicates unlimited right now, so it 
would presumably need a flag, or else maybe a -1 in min.

As for utility. I know some people enjoy waving the dynamite before dropping it.
Similar to waving the skip flag before activating.

Also, we could certainly make dynamite inherit something from that. maybe a bit 
less of a dX component or something.

```

Original comment by `kyberneticist@gmail.com` on 5 Nov 2012 at 10:33

**Comment by GoogleCodeExporter** [comment]
```
Probably applies to Cake and Hammer too

```

Original comment by `kyberneticist@gmail.com` on 5 Nov 2012 at 10:35

**Comment by GoogleCodeExporter** [comment]
```
Another useful purpose for specifying angle on gears that don't use it.  
Preparing your angle for next turn.  Might matter a bit on time critical games.

```

Original comment by `kyberneticist@gmail.com` on 6 Nov 2012 at 3:08

**Comment by GoogleCodeExporter** [comment]
```
No, initially it was fixed angle.
```

Original comment by `unC0Rr` on 6 Nov 2012 at 9:09

**Comment by GoogleCodeExporter** [comment]
```
Just my opinion, but I quite like the current behavior. It's nice feature to be 
able to wave weapons around. It's not uncommon to see a hog threateningly 
waving a hammer around as he stalks up to a hog waiting on a bridge, for 
example.
```

Original comment by `RedGrin...@gmail.com` on 13 Mar 2013 at 11:21

**Comment by GoogleCodeExporter** [comment]
Original comment by `vittorio...@gmail.com` on 4 Jun 2013 at 2:09
* Changed state: **ClosingIfNoActivity**

**Comment by GoogleCodeExporter** [comment]
```
So any decision on this? can we close it as WONTFIX?
```

Original comment by `vittorio...@gmail.com` on 26 Jun 2013 at 7:30

**Comment by GoogleCodeExporter** [comment]
Original comment by `unC0Rr` on 16 Aug 2013 at 9:44
* Changed state: **WontFix**
